app/app.go

```diff
@@ -134,7 +134,8 @@ func (a *App) HTMLTemplates() *template.Template {
 }
 
 func (a *App) Handle404(w http.ResponseWriter, r *http.Request) {
-	mlog.Debug(fmt.Sprintf("%v: code=404 ip=%v", r.URL.Path, utils.GetIpAddress(r)))
+	ipAddress := utils.GetIpAddress(r, a.Config().ServiceSettings.TrustedProxyIPHeader)
+	mlog.Debug(fmt.Sprintf("%v: code=404 ip=%v", r.URL.Path, ipAddress))
```

Comment:
Can we fix this to use structured logging?

Reply:
Yeah, good call. Should we potentially open a campaign to fix those? I don't know.